src/rest/QueryResults.ts

```diff
@@ -68,19 +68,27 @@ export interface IQueryResults {
    */
   totalCountFiltered: number;
   /**
-   * The duration of the query on the Coveo platform.
+   * The total duration of the query roundtrip as seen by the browser/client.
    */
   duration: number;
   /**
-   * The duration of the query on the Coveo Index.
+   * The duration of the query spent on the Coveo Index.
```

Comment:
Maybe we could rephrase the `duration` description as follows:

The total query duration, which is the sum of the `indexDuration` and `searchAPIDuration`.

Consequently, we would rephrase the `indexDuration` description like this:

The part of the total query duration that was spent in the Coveo index.

And we could do the same thing for the `searchAPIDuration` description:

The part of the total query duration that was spent in the Coveo Search API.

Reply:
Total query duration is the sum of index duration + search API duration + all network hops.

`Duration > (indexDuration + searchAPIDuration)`

Reply:
Then `duration` could be:

> The total query duration, which is the sum of the `indexDuration` and `searchAPIDuration`, including any latency incurred through required network hops.